src/js/pages/jobs/JobDetailPage.js

```diff
@@ -143,22 +143,27 @@ class JobDetailPage extends mixin(StoreMixin, TabsMixin) {
   }
 
   getActionButtons() {
+    let job = MetronomeStore.getJob(this.props.params.id);
+
     let dropdownItems = [
       {
         className: 'hidden',
         html: 'More',
         id: JobActionItem.MORE
       },
       {
-        html: 'Disable Schedule',
-        id: JobActionItem.SUSPEND
-      },
-      {
         html: 'Destroy',
         id: JobActionItem.DESTROY
       }
     ];
 
+    if (job.getSchedules().length > 0) {
+      dropdownItems.splice(1, 0, {
```

Comment:
I'd rather push the items in the right order, then using splice to and one in the middle...